addons/mail/static/src/components/activity_mark_done_popover/tests/activity_mark_done_popover_tests.js

```diff
@@ -41,14 +42,26 @@ QUnit.module('activity_mark_done_popover_tests.js', {
 QUnit.test('activity mark done popover simplest layout', async function (assert) {
     assert.expect(6);
 
-    await this.start();
-    const activity = this.messaging.models['Activity'].create({
-        canWrite: true,
-        category: 'not_upload_file',
+    this.data['res.partner'].records.push({
+        activity_ids: [12],
+        id: 100,
+    });
+    this.data['mail.activity'].records.push({
+        activity_category: 'not_upload_file',
+        can_write: true,
         id: 12,
-        thread: insert({ id: 42, model: 'res.partner' }),
+        res_id: 100,
+        res_model: 'res.partner',
+    });
+    const { createChatterContainerComponent } = await this.start();
+    await createChatterContainerComponent({
+        threadId: 100,
+        threadModel: 'res.partner',
+    });
+
+    await afterNextRender(() => {
+        document.querySelector('.o_Activity_markDoneButton').click();
```

Comment:
I've recently added a new `click()` on `start` res to simplify these, those on which you have diff would be nice to use it.

Reply:
Alright !